background/get-addon-settings.js

```diff
@@ -53,10 +53,21 @@ const updatePresetIfMatching = (settings, version, oldPreset = null, presetOrFn
     settings._version = version;
     if (presetOrFn === null) return;
     const map = {};
+    console.groupCollapsed("updatePresetIfMatching log");
     for (const key of Object.keys(oldPreset)) {
-      if (!areSettingsEqual(settings[key], oldPreset[key])) return console.log(settings, oldPreset, key);
+      if (!areSettingsEqual(settings[key], oldPreset[key])) {
+        console.log(`Did not match old preset (version ${version})`, {
+          settingsSnapshot: JSON.parse(JSON.stringify(settings)),
+          checkedAgainst: oldPreset,
+          keyFailed: key,
+        });
+        console.groupEnd();
```

Comment:
This is probably a mistake - there's another call to `groupEnd()` a few lines below this.

Reply:
Note that line 65 is `return;` so the lines below you are mentioning aren't actually being executed.

Reply:
I still don't think I understand why a group needs to be made. It doesn't look like there is ever more than one message in one of these groups.
![Screenshot of console](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/35cba961-bf23-427e-a6a6-31bc039c78a0)

Reply:
I'll just reverse the console log changes before merging